worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/brush/Brush.java

```diff
@@ -40,4 +42,15 @@
      */
     public void build(EditSession editSession, Vector position, Pattern pattern, double size) throws MaxChangedBlocksException;
 
+    /**
+     * Get the region that represents the area that would be affected if called at the given position.
+     *
+     * @param session the editsession to build in
+     * @param position the brush target
+     * @param size the brush size
+     * @return the bounds of the affected area
+     */
+    default public Region getBounds(EditSession session, Vector position, double size) {
```

Comment:
1.12 doesn't guarantee Java 8

Reply:
this isn't really meant to go to 1.12. just using it as a test bed.
also the idea is that third-party brushes will stay compatible, not 100% sure how that works though.